docs/conf.py

```diff
@@ -1,7 +1,12 @@
 # icalendar documentation build configuration file
-import importlib.metadata
 import datetime
-import os
+import importlib.metadata
+import sys
+from pathlib import Path
+
+HERE = Path(__file__).parent
+SRC = HERE.parent / "src"
+sys.path.insert(0, str(SRC))  # update docs from icalendar source for livehtml
```

Comment:
These seems over complicated, and I'm not clear what problem it is supposed to solve. Can you explain?